src/components/common/CommentEditor.js

```diff
@@ -214,6 +219,11 @@ class CommentEditor extends Component {
     adaptStyle();
   };
 
+  handleExpandComment = () => {
+    const { expanded: previousExpanded } = this.state;
+    this.setState({ expanded: !previousExpanded });
```

Comment:
I recommend you use the `prevState` property in `this.setState` and encapsulate everything in one call. This takes care of race conditions.